docker-compose.yaml

```
@@ -1,5 +1,53 @@
+      dockerfile: services/gateway/Dockerfile
+    ports:
+      - 8080:8080
+
+      
+    restart: always
+    command: ["/app/main"]
+
+
+  redis:
+    image: "redis:8-alpine"
+    ports: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)